kits/AGENT/agent-failure-investigator/index.html

```
@@ -0,0 +1,164 @@
+            </div>
+          </div>
+
+          <div class="field-title">Evidence</div>
+          <div id="evidence"></div>
+
+          <div class="field-title">Root cause</div>
+          <p id="root-cause"></p>
+          <div class="mode-tag" id="root-cause-mode"></div>
+          <div class="llm-row">
+            <input id="api-key" type="password" placeholder="Optional: Anthropic API key for LLM narration" autocomplete="off" />
+            <button id="llm-btn" class="ghost">Compose with Claude</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)